pontoon/sync/formats/xliff.py

```diff
@@ -152,15 +152,13 @@ def save(self, locale):
             "nn-NO": "nn",
             "sv-SE": "sv",
         }
-        locale_code = locale.code
-        if locale_code in locale_mapping:
-            locale_code = locale_mapping[locale_code]
+
+        locale_code = locale_mapping.get(locale.code, locale.code)
 
         # Set target-language if not set
         file_node = self.xliff_file.namespaced("file")
         for node in self.xliff_file.document.getroot().iterchildren(file_node):
-            if not node.get("target-language"):
```

Comment:
We don't want to always set a target-language, because then we're back to https://bugzilla.mozilla.org/show_bug.cgi?id=1713058

I know it's hack-ish, but could we do something like this instead?

```
if node.get("target-language") not in ["en", "en-US"]
```

Reply:
Actually, we would still need to check if the value is set, not sure if this works in lxml-land

```
if node.get("target-language", "en-US") not in ["en", "en-US"]:
```